packages/design-system/src/components/BaseLink/BaseLink.tsx

```diff
@@ -1,31 +1,43 @@
 import * as React from 'react';
 
+import { styled } from 'styled-components';
+
 import { forwardRef } from '../../utilities/forwardRef';
-import { Box, BoxProps } from '../Box';
+import { Box, BoxComponent, BoxProps } from '../Box';
 
 type BaseLinkProps<C extends React.ElementType = 'a'> = BoxProps<C> & {
   disabled?: boolean;
   isExternal?: boolean;
 };
 
-const BaseLink = forwardRef<HTMLAnchorElement, BaseLinkProps>(
-  ({ href, rel = 'noreferrer noopener', target = '_self', disabled = false, isExternal = false, ...props }, ref) => {
+const BaseLinkImpl = forwardRef<HTMLAnchorElement, BaseLinkProps>(
+  ({ href, disabled = false, isExternal = false, ...props }, ref) => {
     return (
-      <Box
+      <BaseLink
         tag="a"
         ref={ref}
-        target={isExternal ? '_blank' : target}
-        rel={isExternal ? rel : undefined}
-        href={disabled ? '#' : href}
+        target={isExternal ? '_blank' : undefined}
+        rel={isExternal ? 'noreferrer noopener' : undefined}
+        href={href}
+        tabIndex={disabled ? -1 : undefined}
         aria-disabled={disabled}
-        cursor="pointer"
+        pointerEvents={disabled ? 'none' : undefined}
+        cursor={disabled ? undefined : 'pointer'}
         {...props}
       />
     );
   },
 );
 
+const BaseLink = styled<BoxComponent<'a'>>(Box)`
+  text-decoration: none;
+
+  &:visited {
+    color: inherit;
```

Comment:
Which color would be inherited exactly? Is it the browser's default or one from our DS?

Reply:
the DS, so if the color is `primary600` it'll stay as that 👍🏼